02-project-intro.Rmd

```diff
@@ -163,6 +163,13 @@ Now we have a good directory structure we will now place/save the data file in t
 >
 > We will load and inspect these data later.
 
+> ## Challenge 2 {.challenge}
+> It is usefull to get some general idea about the dataset, directly from the
```

Comment:
sp. usefull / useful.

This challenge is only valid if users have shell knowledge beforehand. I think the idea of looking at the data outside of R is a valid one however. Perhaps it could be rephrased "Examine the data without using the R-environment". This would allow use of notepad, excel, explorer etc.